cryosparc2/protocols/protocol_cryorefine.py

```diff
@@ -437,13 +433,13 @@ def doRunRefine(self):
         """
         :return:
         """
-        if self.mask is not None:
-            input_group_connect = {"particles": str(self.par),
-                                   "volume": str(self.vol),
-                                   "mask": str(self.mask)}
+        if self.mask.get() is not None:
```

Comment:
This block of code seems repeated many times. Is there a way to refactor this to a common function or similar?